### PR TITLE
WIP-Chimp Cucumber smoke tests

### PR DESCRIPTION
**Pull request by emilioward**
### What is the context of this PR?
This pull request is for the chimp cucumber smoke tests trello card. A basic test has been added to simply open the author tool and check if the title matches the feature file. #

### How to review 
Run **./node_modules/.bin/chimp** in the terminal to see if the checks pass, also check if travis runs the script

### CODE REVIEW

**Comment by samiwel** [comment]
Build failed because it couldn't find `chimp`. You might need to modify the `.travis.yml` file to globally install chimp JS. See https://docs.travis-ci.com/user/languages/javascript-with-nodejs/ for an example. See the Ember Apps example, in particular. They seem to have an `install:` section (which gets run by Travis before after `before_install` and before `scripts`  - see https://docs.travis-ci.com/user/customizing-the-build/ for details). Seems like the node dependencies can be installed during `install` and run during `script`.

File: .travis.yml [hunk 1]
```diff
@@ -2,9 +2,14 @@ language: node_js
 node_js:
   - "7"
 
+install:
+  - npm install java
```
Comment (samiwel) [inline .travis.yml]: Do we need to install java, seems like a big dependency to pull in, but Cucumber might require it - not sure.
If yes, then you'll probably need to [install dependencies e](https://docs.travis-ci.com/user/installing-dependencies/).g.

```
before_install:
  - sudo apt-get -qq update
  - sudo apt-get install -y java
```

**Comment by hamishtaplin** [comment]
@emilioward @samiwel can we use `yarn` instead of `npm` to install stuff on Travis?

**Comment by yoganandkunche** [comment]
[Add smoke tests (5)](https://trello.com/c/ltEZIBdG/26-add-smoke-tests-5)

File: .travis.yml [hunk 2]
```diff
@@ -3,16 +3,14 @@ node_js:
...
   - yarn add chimp
   - yarn install
 
 script:
   - yarn lint
   - yarn test -- --coverage
   - yarn build
-  - yarn smoketest
-
+  - ./node_modules/.bin/chimp
```
Comment (WickyNilliams) [inline .travis.yml]: `yarn smoketest` will run this exact command, no need to replace it

Comment (emilioward) [inline .travis.yml]: cool I've added it back in

**Pull request closed without merging**